aimmo-game-worker/test-turn.py

```diff
@@ -8,17 +8,14 @@
     'avatar_state': avatar,
     'world_map': {
         'cells': [
-            {'location': {'x': 0, 'y': 0}, 'habitable': True,
-                'generates_score': True, 'avatar': avatar, 'pickup': None},
-            {'location': {'x': 1, 'y': 0}, 'habitable': False,
-                'generates_score': False, 'avatar': None, 'pickup': None},
-            {'location': {'x': -1, 'y': 0}, 'habitable': True, 'generates_score':
-                False, 'avatar': avatar, 'pickup': {'health_restored': 3}},
+            {'location': {'x': 0, 'y': 0}, 'habitable': True, 'avatar': avatar, 'pickup': None},
+            {'location': {'x': 1, 'y': 0}, 'habitable': False, 'avatar': None, 'pickup': None},
+            {'location': {'x': -1, 'y': 0}, 'habitable': True, 'avatar': avatar, 'pickup': {'health_restored': 3}},
         ],
     }
 }
 
-print 'Posting:', api_data
+print('Posting:', api_data)
```

Comment:
Not necessarily to the PR, maybe we should use logs here and move the package to the tests folder.